src/assets/wise5/classroomMonitor/classroomMonitorComponents/nodeProgress/nav-item/nav-item.component.ts

```diff
@@ -32,117 +32,116 @@ class NavItemController {
   maxScore: number;
   newAlert: boolean = false;
   nodeHasWork: boolean;
-  nodeId: string;
+  @Input() nodeId: string;
   nodeTitle: string;
+  @Output() onExpandedEvent: EventEmitter<any> = new EventEmitter();
   parentGroupId: string = null;
   rubricIconClass: string;
   rubricIconLabel: string;
   rubricIconName: string;
-  showPosition: any;
+  @Input() showPosition: any;
```

Comment:
Change variable type to boolean? Is there as case when this is ever set to false?